lti_consumer/lti_xblock.py

```diff
@@ -710,6 +700,14 @@ def editable_fields(self):
         if not is_external_config_filter_enabled:
             noneditable_fields.append('external_config')
 
+        if self.lti_version == 'lti_1p3' and is_database_config_enabled and self.config_type == 'database':
```

Comment:
@MichaelRoytman I believe this would be leftover code from #260. I am not sure why rebasing hasn't updated it. Maybe a conflict line. Let me update it to the latest code.